docs/conf.py

```diff
@@ -61,6 +61,10 @@
     ("py:class", "SimTime"),
     ("py:class", "immutables._map.Map"),
     ("py:class", "_asyncio.Task"),
+    ("py:class", "tickit.devices.cryostream.status.Status"),
+    ("py:class", "tickit.devices.cryostream.status.ExtendedStatus"),
+    ("py:class", "optional"),
+    ("py:class", "class"),
```

Comment:
`class` should not be ignored. In-fact, it should never appear in any docstring afaik. The solution is to remove `Args: ...` from any `Class` docstrings which contain them.

Reply:
I did think those were odd but git blame said you did them so I left them there.

Reply:
Ooops... My docstring generator always adds them automatically, must've been in such a rush I filled them out instead of deleting them :rofl:

Reply:
No worries, perhaps I trust you too much though :rofl: